python/publish/publisher.py

```diff
@@ -305,6 +313,14 @@ def publish_json(self, data: PublishData):
         # provide a reduced version to Github actions
         self._gha.set_output('json', json.dumps(data.reduced(), ensure_ascii=False))
 
+    def publish_summary(self, title: str, summary: str):
+        if self._settings.job_summary_file:
+            try:
+                with open(self._settings.job_summary_file, 'a+', encoding='utf-8') as summary_file:
```

Comment:
The `+` in `a+` is no really needed, is it?

Reply:
Force of habits, I always put the plus because it won't hurt (neither add perf overhead afaik), but indeed I can remove it.